options/locale/locale_en-US.ini

```
@@ -521,8 +521,8 @@ issue_assigned.issue = @%[1]s assigned you to issue %[2]s in repository %[3]s.
 
 issue.x_mentioned_you = <b>@%s</b> mentioned you:
 issue.action.force_push = <b>%[1]s</b> force-pushed the <b>%[2]s</b> from %[3]s to %[4]s.
-issue.action.push_1 = <b>@%[1]s</b> pushed %[3]d commit to %[2]s
-issue.action.push_n = <b>@%[1]s</b> pushed %[3]d commits to %[2]s
+issue.action.push_1 = <b>@%[1]s</b> pushed %[3]d commit to %[2]s.
```

Not a sentence.
That is displayed in the issue timeline if I'm not mistaken.

---

Isn't "<span>@</span>dave pushed 1 commit to xyz" a sentence? I agree though that the period is probably not needed if this is displayed in an issue timeline, since those messages are more floating than embedded in other text.

---

Well…
I mean yes, syntactically speaking it is a sentence.
But as you said yourself, in this context it isn't.

```suggestion
issue.action.push_1 = <b>@%[1]s</b> pushed %[3]d commit to %[2]s
```